dk_car_scraper/scraper.py

```diff
@@ -11,12 +10,27 @@
 CAPITALIZED_BRANDS = ['VW', 'BMW']
 
 
+def _get_text_value(content, *args):
+    """Returns the value inside the content"""
+    try:
+        value = content
+        for index in list(args):
+            value = value[index]
+
+        if value != content and type(value) == element.Tag:
+            return True, value.text
+
+    except (IndexError, KeyError, AttributeError):
```

Comment:
Is started thinking that this method could return any type of value (including False, None and that value).